wallet-new/test/unit/Test/Spec/CoinSelection/Generators.hs

```diff
@@ -320,6 +340,18 @@ genPayee _utxo payment = do
             , allowRedeemAddresses  = False
             }
 
+genPayeeWithNM :: NetworkMagic -> Core.Utxo -> Pay -> Gen (NonEmpty Core.TxOut)
+genPayeeWithNM nm _utxo payment = do
+    let balance            = toLovelaces payment
+    genTxOutWithNM nm StakeGenOptions {
```

Comment:
Genuine question: I thought stake distribution wasn't used in cardano addresses?

Reply:
Is it important to get an answer to this?